test-app/package.json

```diff
@@ -90,5 +90,8 @@
   },
   "ember": {
     "edition": "octane"
+  },
+  "volta": {
```

Comment:
I don't want to use volta hahaha please remove it

Reply:
@villander removed, it didn't have to be in commit - it's just smth I use on my local